server/users/testUsers.js

```diff
@@ -7,140 +7,129 @@ exports.createTestUsers = function() {
   User.findOne({
     username: 'johnsmith'
   })
-  .then(function(user) {
-    if (user) {
-    }
-    else {
-      var testUser = new User({
-        "facebookId": "10153623461113905",
-        "profilePhoto": "placeholder/image",
-        "username": "johnsmith",
-        "firstName": "John",
-        "lastName": "Smith",
-        "phoneNumber": 16692269013,
-        "cardNumber": 9901230120130,
-        "cvc": "123",
-        "expirationDate": '11/14',
-        "billingAddress": "Stonegate Drive",
-        "city": "San Francisco",
-        "state": "California",
-        "country": "USA",
-      });
+    .then(function(user) {
+      if (user) {} else {
```

Comment:
I wrote the original code and I want to stab myself for writing such terrible code. That said, can't we just write `if (!user) { ... create user ... }`?